backend/app/Models/Submission.php

```diff
@@ -54,4 +64,27 @@ public function files(): HasMany
     {
         return $this->hasMany(SubmissionFile::class);
     }
+
+    /**
+     * @return string
+     */
+    public function getStatusNameAttribute()
+    {
+        $statuses = [
+            1 => 'Initially Submitted',
```

Comment:
Should these be strings that are easier to match on the front end rather than human readable. Eg. "INITAL_SUBMISSION"

Reply:
This has been refactored as advised. These are now underscore-delimited and in all caps.